pyccel/ast/variable.py

```diff
@@ -199,7 +198,8 @@ def __init__(
         self._order          = order
         self._is_argument    = is_argument
         self._is_kwonly      = is_kwonly
-        super().__init__(name)
+
+        PyccelAstNode.__init__(self)
```

Comment:
This can be changed to `super().__init__()` now that there is no ambiguity between the superclasses